cmd/brew-test-bot.rb

```diff
@@ -1525,13 +1525,14 @@ def test_ci_upload(tap)
       bintray_repo = bottle_hash["bintray"]["repository"]
       bintray_packages_url =
         "https://api.bintray.com/packages/#{bintray_org}/#{bintray_repo}"
+      bintray_bottle_url = "https://dl.bintray.com/#{bintray_org}"
```

Comment:
What about `"https://#{bintray_org}.bintray.com"`?

Reply:
It does not work for everyone.
https://dawidd6.bintray.com/bottles-tap/keybase-cli-4.4.1.x86_64_linux.bottle.tar.gz
vs
https://bintray.com/dawidd6/bottles-tap/keybase-cli-4.4.1.x86_64_linux.bottle.tar.gz

Reply:
I don't want to change the default URL that does work for Homebrew.

Reply:
@MikeMcQuaid Why do you prefer `ORG.bintray.com` over `dl.bintray.com/ORG`? The former only works for Homebrew because we're sponsored. I prefer the latter that works for everyone.

Reply:
It reads nicer to me but mostly:

> I don't want to change the default URL that does work for Homebrew.

Reply:
My understanding is that Homebrew would like to improve the experience for third party taps and bottling on third party taps. This change would is a step toward that goal. One option would be to keep `homebrew.bintray.com` for official taps and use `dl.bintray.com/homebrew` for third party taps.

Reply:
there is also `linuxbrew.bintray.com`